_common/KeymanHosts.php

```diff
@@ -107,12 +109,29 @@ function __construct() {
         $this->keyman_com = "https://keyman-staging.com";
         $this->keymanweb_com = "https://keymanweb.com";
         $this->r_keymanweb_com = "https://r.keymanweb.com";
+      } else if($this->tier == KeymanHosts::TIER_TEST) {
+        $this->s_keyman_com = "https://s.keyman.com";
+        $this->api_keyman_com = "http://host.docker.internal:8098"; // Unique for api.keyman.com
```

Comment:
btw, why 8098 and not 8058?

Reply:
That is, all the other sites are in the range 805x. Only api.keyman.com differs.

Reply:
It's just ports we ended up on after changing several things during EMDC.
I can try 8098 and 8099